src/screens/SignInV2/elements/BottomOverlay/pagination.js

```diff
@@ -30,7 +30,7 @@ const styles = StyleSheet.create({
     return {
       width: 12.63,
       height: 4,
-      backgroundColor: isActive ? COLORS.signed_primary : COLORS.black30percent,
+      backgroundColor: isActive ? COLORS.signed_primary : COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.black30percent` to `COLORS.gray200` seems fine as it's a color change. However, ensure that the new color `COLORS.gray200` is accessible and provides enough contrast for users with visual impairments. If not, consider using a more contrasting color.
```diff
- backgroundColor: isActive ? COLORS.signed_primary : COLORS.black30percent,
+ backgroundColor: isActive ? COLORS.signed_primary : COLORS.gray200,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->